Gemfile

```diff
@@ -1,59 +1,59 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
-gem 'activerecord-session_store', '1.0.0'
-gem 'unicorn', '4.6.2'
-gem 'pg', '0.18.4'
-gem 'optic14n', '2.0.1'     # Ideally version should be synced with bouncer
-gem 'gds-sso', '12.0.0'
-gem 'govuk_admin_template', '4.2.0'
-gem 'plek', '1.12.0'
-gem 'htmlentities', '4.3.4'
-gem 'kaminari', '0.16.3'
+gem 'rails', '5.1.0'
+gem 'activerecord-session_store'
+gem 'unicorn'#, '4.6.2'
```

Comment:
Before you squash all these commits, could you please remove the commented out versions?